pkg/build/gobuild.go

```diff
@@ -272,7 +272,11 @@ func build(ctx context.Context, ip string, platform v1.Platform, disableOptimiza
 	cmd.Stderr = &output
 	cmd.Stdout = &output
 
-	log.Printf("Building %s for %s/%s", ip, platform.OS, platform.Architecture)
+	plat := path.Join(platform.OS, platform.Architecture)
```

Comment:
Do you really want to couple the separator in the label between OS and Architecture based on the path separator ? What happens if you run this on Windows ? And if you have tooling that relies on that log output to be stable (yes, such things happen). Also, it is really not related to a file path.

So I highly recommend using print with a fixed "/" separator. 

Or even just `+`

Reply:
`path` is [fixed](https://golang.org/src/path/path.go#L160) as `"/"`, `filepath` is [based on your current platform](https://godoc.org/path/filepath#Join).

Reply:
ok, thanks for the heads (didnt know that). Fine if you consider the architecture label to be an URL like 'path' (still like the more explicit form better)

Reply:
Sure, done.